lib/filters/validator.js

```diff
@@ -43,14 +43,16 @@ const supportedValidators = ['maximum',
  * @constructor
  */
 class Validator {
-    constructor(parameters, definitions) {
+    constructor(parameters, components) {
         this._ajv = constructAjv({ verbose: true });
-        if (definitions) {
-            Object.keys(definitions).forEach((schemaName) => {
-                this._ajv.addSchema(
-                    definitions[schemaName],
-                    `#/definitions/${schemaName}`
-                );
+        if (components) {
+            Object.keys(components).forEach((key) => {
+                Object.keys(components[key]).forEach((schemaName) => {
```

Comment:
I am not entirely sure... the components can have much more different stuff then just schemas and not all of them will be valid schemas. I think we only need properties that are schemas.

Also, there's now 'components.parameters' - if we start using those I don't think those would work

Reply:
You're right. Done.